client/valor/schemas/symbolic/operators.py

```diff
@@ -1,223 +1,234 @@
-from typing import Any
+import warnings
+from typing import Any, Optional, Union
 
 
-class Function:
-    """Base class for defining a function."""
+class Condition:
+    """Base class for defined a filter condition."""
 
-    _operator = None
+    def __init__(self, lhs: Any, rhs: Optional[Any] = None) -> None:
+        """
+        TODO
```

Comment:
fix TODOs everywhere in the repo? there were some outstanding ones from prior filtering PRs as well

Reply:
Searched for and removed / completed todos.